tests/models/language/generation/test_hybrid.py

```diff
@@ -53,17 +54,21 @@
 ]
 
 V1_SUPPORTED_MODELS = [
-    "mistralai/Mamba-Codestral-7B-v0.1",
-    "ibm-ai-platform/Bamba-9B-v1",
-    "Zyphra/Zamba2-1.2B-instruct",
-    "nvidia/Nemotron-H-8B-Base-8K",
-    "ibm-granite/granite-4.0-tiny-preview",
-    "tiiuae/Falcon-H1-0.5B-Base",
+    "mistralai/Mamba-Codestral-7B-v0.1", "ibm-ai-platform/Bamba-9B-v1",
+    "Zyphra/Zamba2-1.2B-instruct", "nvidia/Nemotron-H-8B-Base-8K",
+    "ibm-granite/granite-4.0-tiny-preview", "tiiuae/Falcon-H1-0.5B-Base",
+    "LiquidAI/LFM2-1.2B"
 ]
 
 # Avoid OOM
 MAX_NUM_SEQS = 4
 
+# To be removed once implemented
+V1_HYBRID_UNSUPPORTED_ARGS = {
+    "enforce_eager": True,
+    "enable_prefix_caching": False,
+}
+
```

Comment:
I don't understand why these need to be added to the test at all. As I mentioned in my previous comment, you are also passing them to the V0 parts of the test which is not what we want. I think `V1_HYBRID_UNSUPPORTED_ARGS ` should be removed completely. The only change to the test needs to be the addition of the new model.